packages/react-badge/src/components/Badge/useBadge.ts

```diff
@@ -26,7 +26,6 @@ export const useBadge_unstable = (props: BadgeProps, ref: React.Ref<HTMLElement>
     },
     root: getNativeElementProps('div', {
       ref,
-      'aria-hidden': true,
```

Comment:
what is the impact of this change for Avatar? That is a `role='img'` with now a text inside of it. It would be good to test with screen readers and see if we need to give some best practices for Avatar usage.

Reply:
Avatar currently has `aria-labelledby` pointing to both itself and the badge, which right now has `aria-hidden`. Where that currently works, we're relying on behavior quirks. It'd be better to remove the `aria-hidden` so the accname for the image is built from non-hidden content.